.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+$finder = (new PhpCsFixer\Finder())
+    ->in([
+        __DIR__ . '/src',
+        __DIR__ . '/inc',
+    ])
+;
+
+return (new PhpCsFixer\Config())
+    ->setParallelConfig(PhpCsFixer\Runner\Parallel\ParallelConfigFactory::detect())
+    ->setCacheFile(sys_get_temp_dir() . '/php-cs-fixer.glpi.cache')
+    ->setRules([
+        '@PER-CS2.0' => true,
+        '@PHP82Migration' => true,
```

Comment:
`@PHP84Migration` include all previous versions migrations.

```suggestion
        '@PHP84Migration' => true,
```